travis/publish_to_bintray.sh

```diff
@@ -4,7 +4,7 @@ set -ev
 
 mvn deploy -Dmaven.test.skip=true -DskipTests -Dscoverage.skip --projects com.yahoo.maha:maha-parent --settings travis/settings-maha-parent.xml
 mvn deploy -Dmaven.test.skip=true -DskipTests -Dscoverage.skip --projects com.yahoo.maha:maha-druid-lookups --settings travis/settings-maha-druid-lookups.xml
-mvn deploy -Dmaven.test.skip=true -DskipTests -Dscoverage.skip --projects com.yahoo.maha:maha-core --settings travis/settings-maha-core.xml
```

Comment:
-Dmaven.test.skip=true -Dscoverage.skip=true -DskipTests -Darguments="-DskipTests -Dscoverage.skip=true" 

add this to all publish statements